frontend/src/modules/studySamples/services.ts

```diff
@@ -125,6 +125,10 @@ export async function buildStudySamplesFromWorkflow(
 			}
 			const samples = sampleNextSteps.map(nextStep => nextStep.sample)
 
+			const sampleCountByPooledSampleID =
```

Comment:
We should avoid calling this endpoint if there are zero samples at the current step (happens often).

Reply:
Also, it's a shame that we are forced to call the endpoint even if there are no pooled samples in the list (since we don't know if the sample is pooled or not).
I know the story was about displaying the number of samples in a pool for a study, but maybe we might want to display the same thing elsewhere. I wonder if it would be better to add a new field to Sample, `pooledSampleCount` and have the samples endpoint return a value in that field if the sample is a pool (and undefined/null if not).

Reply:
We could replace is_pool by an integer pooled_count (or other name) (1 -> x) and provide an is_pool function in the frontend to convert 1 to false and anything more to true... We currently use a count to test for pool in the endpoint but only return a boolean.